effekt/shared/src/main/scala/effekt/core/Renamer.scala

```diff
@@ -81,6 +81,19 @@ class Renamer(names: Names = Names(Map.empty), prefix: String = "") extends core
       }
   }
 
+  override def rewrite(o: Operation): Operation = o match {
+    case Operation(name, tparams, cparams, vparams, bparams, resume, body) =>
+      withBindings(tparams ++ cparams ++ vparams.map(_.id) ++ bparams.map(_.id) ++ resume.map(_.id).toList) {
+        Operation(name,
+          tparams map rewrite,
+          cparams map rewrite,
+          vparams map rewrite,
+          bparams map rewrite,
+          resume map rewrite,
+          rewrite(body))
+      }
+  }
```

Comment:
I needed to fix renamer since we forgot to rename parameters bound by operations.